doc/book/box/atomic/txn_mode_mvcc.rst

```diff
@@ -85,15 +85,15 @@ of successful completion of the transaction and helps to avoid possible conflict
 To set the default isolation level with the other option, for example, 
 to ``read-committed``, use the following command:
 
-..  code-block:: 
+..  code-block:: lua
 
     box.cfg{default_txn_isolation = 'read-committed'}
  
 
```

Comment:
It would be nice if we could learn from this section that the isolation levels can also be set via IPROTO.

Reply:
Thanks, added this.